src/test/java/org/springframework/samples/petclinic/web/VetControllerTests.java

```diff
@@ -14,16 +19,29 @@
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
 
 import static org.hamcrest.xml.HasXPath.hasXPath;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Put all the selenium tests in a different file named VetTest.java, because otherwise it breaks CI. Also, to keep consistent with existing tests, use the ChromeDriver.

Reply:
put them in AppointmentTest, with the other vet profile test